livekit_models.proto

```diff
@@ -25,6 +25,19 @@ message Codec {
   string fmtp_line = 2;
 }
 
+message ParticipantPermission {
+  // allow participant to subscribe to other tracks in the room
+  bool can_subscribe = 1;
+  // allow participant to publish new tracks to room
+  bool can_publish = 2;
+  // allow participant to publish data
+  bool can_publish_data = 3;
+  // indicates that it's hidden to others
+  bool hidden = 7;
```

Comment:
gaps are due to deprecated stuff in between?

Reply:
yup, for backwards compatibility, field numbers cannot change.